src/migration.rs

```diff
@@ -66,11 +88,60 @@ pub fn migrate(from: &Path, mut to: Options) -> Result<()> {
 			}
 			true
 		})?;
+		if overwrite {
+			dest.commit_raw(commit)?;
+			commit = Vec::with_capacity(COMMIT_SIZE);
+			std::mem::drop(dest);
+			dest = Db::open_or_create(&to)?; // This is needed to flush logs.
+			log::info!("Collection migrated {}, imported", c);
+
+			std::mem::drop(dest);
+			std::mem::drop(source);
+			move_column(c, &to.path, from)?;
+			source_options.columns[c as usize] = to.columns[c as usize].clone();
+			source_options.write_metadata(&metadata_path, &to.salt.expect("Migrate requires salt"))
+				.map_err(|e| Error::Migration(format!("Error {:?}\nFail updating metadata of column {:?} \
+							in source, please update manually before restarting.", e, c)))?;
+			source = Db::open(&source_options)?;
+			dest = Db::open_or_create(&to)?;
+
+			log::info!("Collection migrated {}, migrated", c);
+		}
 	}
 	dest.commit_raw(commit)?;
 	Ok(())
 }
 
+fn move_column(c: ColId, from: &Path, to: &Path) -> Result<()> {
+	deplace_column(c, from, to, false)
+}
+
+fn copy_column(c: ColId, from: &Path, to: &Path) -> Result<()> {
+	deplace_column(c, from, to, true)
+}
+
+fn deplace_column(c: ColId, from: &Path, to: &Path, copy: bool) -> Result<()> {
+	for entry in std::fs::read_dir(from)? {
+		let entry = entry?;
+		if let Some(file) = entry.path().file_name().and_then(|f| f.to_str()) {
+			if crate::index::TableId::is_file_name(c, file)
+				|| crate::table::TableId::is_file_name(c, file) {
+
+				let mut from = from.to_path_buf();
+				from.push(file);
+				let mut to = to.to_path_buf();
+				to.push(file);
+				if copy {
+					std::fs::copy(from, to)?;
+				} else {
+					std::fs::rename(from, to)?;
```

Comment:
After #37 source and dest might end up with a different set of tables. So after "rename" some junk tables may be left.

Reply:
:+1: 
Looking at files dates, I did already have one.